examples/kubectl.rs

```diff
@@ -21,33 +21,43 @@ use tracing::*;
 
 #[derive(clap::Parser)]
 struct App {
-    #[clap(long, short, arg_enum, default_value_t)]
+    #[arg(long, short, default_value_t = OutputMode::Pretty)]
     output: OutputMode,
-    #[clap(long, short)]
+    #[arg(long, short)]
     file: Option<std::path::PathBuf>,
-    #[clap(long, short = 'l')]
+    #[arg(long, short = 'l')]
     selector: Option<String>,
-    #[clap(long, short)]
+    #[arg(long, short)]
     namespace: Option<String>,
-    #[clap(long, short = 'A')]
+    #[arg(long, short = 'A')]
     all: bool,
-    #[clap(arg_enum)]
     verb: Verb,
     resource: Option<String>,
     name: Option<String>,
 }
 
-#[derive(clap::ArgEnum, Clone, PartialEq, Eq)]
+#[derive(Clone, PartialEq, Eq, clap::ValueEnum)]
 enum OutputMode {
     Pretty,
     Yaml,
 }
-impl Default for OutputMode {
-    fn default() -> Self {
-        Self::Pretty
+
+impl OutputMode {
+    fn as_str(&self) -> &'static str {
+        match self {
+            Self::Pretty => "pretty",
+            Self::Yaml => "yaml",
+        }
+    }
+}
```

Comment:
oh, this requires more boilerplate now in clap 4 :thinking: 
i guess they want people to just use `Display`, which can be derived in other ways

Reply:
Right, I can fold it into `Display` itself, but I just so used to do it that way in many other places, that fingers typed that automatically

Reply:
it's fine :-)